db/migrate/20210827111245_add_correspondence_language_to_person_by_layer.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2021, Schweizer Blasmusikverband. This file is part of
+#  hitobito_sbv and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito_sbv.
+
+class AddCorrespondenceLanguageToPersonByLayer < ActiveRecord::Migration[6.0]
+  def up
+    people_without_language.find_each do |person|
+      person.correspondence_language = person.primary_group.correspondence_language
+      person.save
+    end
```

Comment:
Das hier ist komplett ohne Ausgabe, solange die Daten umgewandelt werden, was während einem Deployment etwas Unruhe auslösen kann. Da in Produktion eine Menge Leute vorhanden sind, kann das recht lange laufen.

Zumindest sollte `say_with_time` hier verwendet werden, um das Feedback zu verbessern. Abhängig von der tatsächlichen Laufzeit kann es auch hilfreich sein, vielleicht auch Zwischenschritte zu visualisieren (siehe: https://github.com/hitobito/hitobito_die_mitte/blob/master/db/migrate/20210713233600_translate_role_labels.rb#L32-L44).